.github/workflows/python-package.yml

```diff
@@ -0,0 +1,72 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  MJKEY: ${{ secrets.MJKEY }}
+
+jobs:
+#  test:
+#
+#    runs-on: ubuntu-latest
+#
+#    steps:
+#    - uses: actions/checkout@v2
+#    - name: Set up Python 3.5
+#      uses: actions/setup-python@v2
+#      with:
+#        python-version: 3.6
+#
+#    - name: Install mujoco
+#      run: |
+#        sudo apt-get install libosmesa6-dev libgl1-mesa-glx libglfw3
+#        sudo ln -s /usr/lib/x86_64-linux-gnu/libGL.so.1 /usr/lib/x86_64-linux-gnu/libGL.so
+#        curl -O https://www.roboti.us/download/mujoco200_linux.zip
+#        unzip -q mujoco200_linux.zip
+#        mkdir ~/.mujoco && mv mujoco200_linux ~/.mujoco/mujoco200
+#        echo $MJKEY > ~/.mujoco/mjkey.txt
+#
+#    - name: Install setup.py dependencies
+#      run: |
+#        export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:~/.mujoco/mujoco200/bin
+#        python -m pip install --upgrade pip
+#        pip install ".[dev]"
+#
+#    - name: Test with pytest
+#      run: |
+#        export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:~/.mujoco/mujoco200/bin
+#        pytest -n 0 -v -m 'not large and not skip_on_ci'
```

Comment:
can you remove your commented out scripting here?

Reply:
Sure, though it very nearly works and would be much faster than building the full docker container. I was gonna leave it here for whoever has to deal with the CI next time it breaks. Should I just move it to another branch?

Reply:
Yup you could do that. The reason to use docker is because we can control all the dependencies, and they don't change from under us if gh actions changes for whatever reason. We can also use docker to control the resources that are used by the test, and if we ever need to move to another service for running the CI, then we can always just move using the docker image.